pallets/connectors/src/message.rs

```diff
@@ -3,6 +3,17 @@ use sp_std::{vec, vec::Vec};
 
 use crate::*;
 
+/// Address type
+/// Note: It can be used to represent any address type with a length <= 32 bytes;
+/// For example, it can represent an Ethereum address (20-bytes long) by padding it with 12 zeros.
+type Address = [u8; 32];
+
+/// The fixed size for the array representing a tranche token name
+pub const TOKEN_NAME_SIZE: usize = 128;
+
+// The fixed size for the array representing a tranche token symbol
+pub const TOKEN_SYMBOL_SIZE: usize = 32;
```

Comment:
Why not use the constants defined in libs crate?

Reply:
I guess we can do that. The reason I didn't go that route is that the other constants are a `max` length for tranche name and symbols on the pools crates, while this has to do with the  fixed `size` in bytes for the name and symbol take in the `AddTranche` message. And we don't want someone to change the constants in the libs crate and have connectors still compile; but as I write this, the tests will pick that up so that should be fine.

Reply:
Yeah, I thought a single source of truth would be great and the test in connectors fail if we change the constants in the lib that both connectors and pools uses. Is this what you meant? ^^

Reply:
Nevermind! Just saw the test. That is perfect!